src/Analyzers/Microsoft.Analyzers.Extra/CallAnalysis/Fixers/LegacyLoggingFixer.cs

```diff
@@ -335,7 +335,7 @@ private static string GetNamespace(ClassDeclarationSyntax cl)
     /// <summary>
     /// Given a LoggerExtensions method invocation, produce a parameter list for the corresponding generated logging method.
     /// </summary>
-    private static IReadOnlyList<SyntaxNode> MakeParameterList(
+    private static List<SyntaxNode> MakeParameterList(
```

Comment:
mmm not sure I understand how this change is related.

Reply:
Analyzer warning that popped out when fiddling with things.